python-modules/chevah-python-test/test_python_binary_dist.py

```diff
@@ -421,7 +421,7 @@ def main():
 
             if chevah_os == 'windows':
                 # Check OpenSSL version on windows.
-                expecting = u'OpenSSL 1.0.2g  1 Mar 2016'
+                expecting = u'OpenSSL 1.1.0f  25 May 2017'
```

Comment:
Great change.

We will need a follow up ticket on Windows for chevah/server in which we will update paver.conf to the to be release python-package version

and then update the SFTPPlus documentation to inform that on Windows we have OpenSSL 1.1.0